app/lib/Plugins/SearchEngine/Elastic8/dynamicTemplates.json

```diff
@@ -63,8 +63,15 @@
       "match_mapping_type" : "*",
       "match" : "*-kw",
       "mapping" : {
-        "type" : "keyword",
-        "copy_to" : "_all"
+        "type" : "text",
```

Comment:
I'm not comfortable with this. Keywords should be keywords because ES will then use these for searching rather than a full text search. It would probably be safer to add the "ignore_above" in here as well. If we want to do partial matches then we _can_ use wildcard type fields instead.

Reply:
Happy to yank this :+1: